src/Perlang.Tests.Integration/Operator/Comparison.cs

```diff
@@ -261,9 +263,12 @@ public void zero_and_negative_zero_integers_are_identical()
             Assert.Equal(false, output);
         }
 
-        [Fact]
-        public void zero_less_than_negative_zero_is_false()
+        [Theory]
+        [ClassData(typeof(TestCultures))]
+        public void zero_less_than_negative_zero_is_false(CultureInfo cultureInfo)
```

Comment:
Oops, this illustrates the problem mentioned in https://github.com/perlang-org/perlang/pull/264/files#r793446103. :see_no_evil: It's very easy to make these kind of mistakes by accident.